packages/framework/src/Migrations/AbstractMigration.php

```diff
@@ -0,0 +1,72 @@
+<?php
```

Comment:
I would update commit message to something like:
`replaced MultidomainMigrationTrait trait by AbstractMigration class`

Reply:
Also those two commits should be probably squashed as now the first commit violates YAGNI and use is added in second one.